build/ci/evergreen.yml

```diff
@@ -259,15 +259,15 @@ tasks:
     must_have_test_results: true
```

Comment:
let's now make it required to see if it separates the tasks tags: ["e2e","required"]

Reply:
in line 258